public/nativedisplay.h

```diff
@@ -65,6 +65,7 @@ class NativeDisplay {
 
   virtual bool GetDisplayConfigs(uint32_t *num_configs, uint32_t *configs) = 0;
   virtual bool GetDisplayName(uint32_t *size, char *name) = 0;
+  virtual bool GetDisplayPipe(uint32_t *pipe_id) = 0;
```

Comment:
Can  u add comments for this API, instead of boolean u can return int, if it returns -1 than we dont have display connected.